Update README.md (#2)
RAKWireless/rakwireless-docs: docs/Product-Categories/WisLink/RAK2247/Quickstart/README.md

Again, the values are just an example. Remember to do this for all 3 files in step 4.

> 5. Change the `global_conf.json` that will be copied during instalaton by replacing the `global_conf.eu_863_870.json` in the end of the (install.sh file)[https://github.com/RAKWireless/rak_common_for_gateway/blob/master/lora/rak2247_usb/install.sh] to one of those inside `/global_conf` or a custom one. (**EU868 is the default**)

Reviewer: ```suggestion
5. Change the `global_conf.json` that will be copied during installation by replacing the `global_conf.eu_863_870.json` in the end of the (install.sh file)[https://github.com/RAKWireless/rak_common_for_gateway/blob/master/lora/rak2247_usb/install.sh] to one of those inside `/global_conf` or a custom one. (**EU868 is the default**)
```

```sh
cp global_conf/global_conf.eu_863_870.json $INSTALL_DIR/packet_forwarder/lora_pkt_fwd/global_conf.json
```

:::tip 📝 NOTE:

>  You may also coment the line below so the TTN address doen't get replaced by localhost.

Reviewer: ```suggestion
 You may also comment on the line below so the TTN address doesn't get replaced by localhost.
```

:::
```sh
# sed -i "s/^.*server_address.*$/\t\"server_address\": \"127.0.0.1\",/" $INSTALL_DIR/packet_forwarder/lora_pkt_fwd/global_conf.json
```

> 6. If you are using the RAK2247 in a board that have diferent pinout than the RAK2247 Pi Hat, replace the `SX1301_RESET_BCM_PIN` in the `rak_common_for_gateway/lora/start.sh` to the corresponding RESET pin.

Reviewer: ```suggestion
6. If you are using the RAK2247 in a board that has a different pinout than the RAK2247 Pi Hat, replace the `SX1301_RESET_BCM_PIN` in the `rak_common_for_gateway/lora/start.sh` to the corresponding RESET pin.
```

```sh
# Reset iC880a PIN
SX1301_RESET_BCM_PIN=<YOUR_RESET_PIN_HERE>
```
:::tip 📝 NOTE:

>  If you want to have your Gateway_ID automaticaly update when running your package fowarder, uncoment and change the line `#./update_gwid.sh ./local_conf.json` in the same file (start.sh) to `./update_gwid.sh ./global_conf.json`.

Reviewer: ```suggestion
 If you want to have your Gateway_ID automatically update when running your package forwarder, uncomment and change the line `#./update_gwid.sh ./local_conf.json` in the same file (start.sh) to `./update_gwid.sh ./global_conf.json`.
```

systemctl enable ttn-gateway.service
```

8. Save “**install.sh**” file and execute it in order to install: 

```sh
sudo ./install.sh
```

> 9.Wait for the installation to complete. Using the commands below, go and run the newly created process (**lora_pkt_fwd**):

Reviewer: ```suggestion
9. Wait for the installation to complete. Using the commands below, go and run the newly created process (**lora_pkt_fwd**):
```

 If you added the additional lines in step 5 it will execute every time on boot.
:::

> 10. To check if it is working, run `sudo systemctl start ttn-gateway.service` to start the service and check its status `service ttn-gateway status`. You should see something like the box bellow. Be aware that it may take some minutes to see your gateway as connected in TTN's console.

Reviewer: ```suggestion
10. To check if it is working, run `sudo systemctl start ttn-gateway.service` to start the service and check its status `service ttn-gateway status`. You should see something like the box below. Be aware that it may take some minutes to see your gateway as connected in TTN's console.
```

   Active: active (running) since Fri 2020-10-09 17:50:55 BST; 1min 9s ago
 Main PID: 721 (start.sh)
    Tasks: 6 (limit: 2065)
   CGroup: /system.slice/ttn-gateway.service
           ├─721 /bin/bash /opt/ttn-gateway/packet_forwarder/lora_pkt_fwd/start.sh
           └─769 ./lora_pkt_fwd

Oct 09 17:52:00 raspberrypi ttn-gateway[721]: src/jitqueue.c:448:jit_print_queue(): INFO: [jit] queue is empty
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: ### [GPS] ###
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: # GPS sync is disabled
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: ##### END #####
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: JSON up: {"stat":{"time":"2020-10-09 16:51:30 GMT","rxnb":0,"rxok":0,"rxfw":0,"ackr":0.0,"dwnb":0,"t
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: INFO: [down] PULL_ACK received in 368 ms
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: ##### 2020-10-09 16:52:00 GMT #####
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: ### [UPSTREAM] ###
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: # RF packets received by concentrator: 0
Oct 09 17:52:00 raspberrypi ttn-gateway[721]: # CRC_OK: 0.00%, CRC_FAIL: 0.00%, NO_CRC: 0.00%

> 

Reviewer: ```suggestion
```
```

> 

Reviewer: Close the code block with ```

Author: Just did the spelling corrections and closed the code block. Thank you ;p

Author: BTW, I totally forgot to use the _spelling checker tool_ when I wrote this 🤦
